### PR TITLE
[FIX] account: Automatic Entries wizard on invoice with date in the past

### PR DESCRIPTION
**Pull request by momegahed**
Steps to reproduce:
1. Create an invoice with an Invoice Date in the past
2. Confirm the invoice
3. In Journal Items tab, click cut-off and then create journal entries
4. Traceback

opw-3369709

### CODE REVIEW

File: addons/account/tests/test_account_move_out_invoice.py [hunk 1]
```diff
@@ -2735,6 +2735,51 @@ def test_out_invoice_change_period_accrual_1(self):
...
+        wizard = self.env['account.automatic.entry.wizard'] \
+            .with_context(context) \
+            .create({
+                'action': 'change_period',
+                'journal_id': self.company_data['default_journal_misc'],
+                'revenue_accrual_account': self.env['account.account'].create({
+                    'name': 'Accrual Revenue Account',
+                    'code': '765432',
+                    'account_type': 'expense',
+                    'reconcile': True,
```
Comment (h4818) [inline addons/account/tests/test_account_move_out_invoice.py:2768]: could you use an existing account?
Or just duplicate the default_revenue account
(Accrual Revenue Account should not be of type `expense` right? and does it need to be reconcilable?

**Comment by antoine162** [comment]
@momegahed thank you so much for this fix and thank you @h4818 for reviewing, to me it looks good :smile:

**Review by h4818 — APPROVED** [review]
Looks good, thanks for the fix @momegahed

**Comment by h4818** [comment]
@robodoo r+

**Pull request closed without merging**